string model_name
uint32[] random_string
---
string model_name
uint32[] random_string
